cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -517,7 +517,7 @@ share
   -- For now they are grouped under the specific hash of the pool.
   ReservedPoolTicker
     name                Text
-    poolId              PoolHashId
+    poolHash            ByteString          sqltype=hash28type
```

Comment:
Why is this? The pool hash can still be obtained via a `join` and using the index adds a small amount of consistency checking.